vue-components/src/components/DateInput.vue

```diff
@@ -92,19 +92,19 @@ export default defineComponent( {
 .wikit-DateInput {
 	&__IntroText {
 		padding-block-end: $wikit-InputExtender-content-padding-vertical;
-		font-family: $font-family-style-label;
-		font-weight: $font-weight-style-label;
-		font-size: $font-size-style-label;
-		line-height: $font-line-height-style-label;
+		font-family: $font-family-style-body;
```

Comment:
Probably worth replacing by component tokens (new json file should be created) to prevent the same issue from happening in a hypothetical future.

Reply:
done in [23c5fe6](https://github.com/wmde/wikit/pull/467/commits/23c5fe64ee7d183a8f1627c636404fecc0a3e059)